block_0001/Shakesneer-20191026180317.xml

```diff
@@ -3,16 +3,18 @@
   <date>2019-10-26T18:03:17Z</date>
   <link>/r/TheMotte/comments/dkvpb6/culture_war_roundup_for_the_week_of_october_21/f5asqv6/</link>
   <title>Kanye West and Sincerity</title>
+  <context>
+    <p>I have a healthy mix of left and right Facebook friends and Twitter follows. It makes it pretty interesting to compare the reactions to culture war events.</p>
+    <p>There has been a pretty stark contrast in reaction to Kanye’s new album which dropped this week: Jesus is King. It is quite explicitly Christian.</p>
+  </context>
```

Comment:
I feel like this needs more context, I had to go to the original post to figure out what was going on.